saveSvgAsPng.js

```diff
@@ -127,6 +127,14 @@
 
               if (externalFontUrl) {
                 // okay, we are lucky. We can fetch this font later
+
+                //handle url if relative
+                if (externalFontUrl.startsWith('../')) {
+                  externalFontUrl = sheets[i].href + '/../' + externalFontUrl
+                } else if (externalFontUrl.startsWith('./')) {
+                  externalFontUrl = sheets[i].href + '/.' + externalFontUrl
```

Comment:
Yes, i mean it.

 ```sheets[i].href``` here is the absolute path of css. Take ```bootstrap``` in my project as example,
 ```sheets[i].href``` is ```file://{projectpath}/node_modules/bootstrap/dist/css/bootstrap.min.css```, then 
```
@font-face {
  font-family: 'Glyphicons Halflings';

  src: url('../fonts/glyphicons-halflings-regular.eot');
 ...
}
``` 
in the file provides externalFontUrl ```../fonts/glyphicons-halflings-regular.eot```.

so the resolved url is ```file://{projectpath}/node_modules/bootstrap/dist/css/bootstrap.min.css/../../fonts/glyphicons-halflings-regular.eot```, which equals to ```file://{projectpath}/node_modules/bootstrap/dist/fonts/glyphicons-halflings-regular.eot``` (using path.normalize() in NodeJS can easily prove it)

In short, because file name is included in ```sheets[i].href``` (```bootstrap.min.css``` in this example), '/../' or '/.' is appended to form  a ```/../``` to skip the file name.

It is not so readable and I don't know if it is more readable to code like this:
 ```
if(externalFontUrl.startsWith('../') || externalFontUrl.startsWith('./')) {// maybe externalFontUrl.startsWith('.') instead
    let cssPath = sheets[i].href
    externalFontUrl = cssPath.substring(0, cssPath.lastIndexOf('/') + 1) + externalFontUrl
}
```
The PR is editable for you.